sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -138,7 +138,7 @@ SyncPoller<CertificateOperation, KeyVaultCertificateWithPolicy> certificatePolle
     certificateClient.beginCreateCertificate("certificateName", CertificatePolicy.getDefault());
 certificatePoller.waitUntil(LongRunningOperationStatus.SUCCESSFULLY_COMPLETED);
 KeyVaultCertificate certificate = certificatePoller.getFinalResult();
-System.out.printf("Certificate created with name \"%s\"\n", certificate.getName());
+System.out.println("Certificate created with name: " + certificate.getName());
```

Comment:
Also, a nit I guess but we do follow `printf()` pattern for samples in other libraries.

Reply:
I see, then we should be changing all our READMES and samples to follow this pattern at some point.